[2020-06-18 14:28:45,081] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: sh_macerado_v3.save_raw 2020-06-18T17:32:17.618288+00:00 [queued]>
[2020-06-18 14:28:45,164] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: sh_macerado_v3.save_raw 2020-06-18T17:32:17.618288+00:00 [queued]>
[2020-06-18 14:28:45,164] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 14:28:45,164] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-18 14:28:45,164] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-18 14:28:45,223] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): save_raw> on 2020-06-18T17:32:17.618288+00:00
[2020-06-18 14:28:45,228] {standard_task_runner.py:53} INFO - Started process 28578 to run task
[2020-06-18 14:28:45,400] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: sh_macerado_v3.save_raw 2020-06-18T17:32:17.618288+00:00 [running]> strx
[2020-06-18 14:28:45,435] {logging_mixin.py:112} INFO - Saving raw...
[2020-06-18 14:28:45,454] {python_operator.py:114} INFO - Done. Returned value was: save_raw_file
[2020-06-18 14:28:45,567] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=sh_macerado_v3, task_id=save_raw, execution_date=20200618T173217, start_date=20200618T182845, end_date=20200618T182845
[2020-06-18 14:28:55,025] {logging_mixin.py:112} INFO - [2020-06-18 14:28:55,025] {local_task_job.py:103} INFO - Task exited with return code 0
